staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -116,8 +119,51 @@ service RuntimeService {
     rpc Status(StatusRequest) returns (StatusResponse) {}
 }
 
+// GetEventsFilter is used to filter a list of events.
+// All those fields are combined with 'AND'
+message GetEventsFilter {
+    // ID of the container, sandbox or image.
+    string id = 1;
+    // LabelSelector to select matches.
+    // Only api.MatchLabels is supported for now and the requirements
+    // are ANDed. MatchExpressions is not supported yet.
+    map<string, string> label_selector = 3;
+}
+
+message GetEventsRequest {
+    // GetEventsFilter to filter a list of events.
+    GetEventsFilter filter = 1;
```

Comment:
optional?

Reply:
Thanks. Yes, it's optional. I will update the comment to reflect that it's optional.